app.rb

```diff
@@ -0,0 +1,25 @@
+require "sinatra/base"
+require "./lib/peep"
+
+class Chitter < Sinatra::Base
+  get "/" do
+    @peep = Peep.all
+    erb :index
+  end
+
+  post "/peeps" do
+    @peep = Peep.add_peep
+    erb :peeps
```

Comment:
It's good practice to redirect after POSTs. Otherwise, there is a risk of someone refreshing the request and sending it twice.